src/components/layout/NotificationDropdown.tsx

```
@@ -14,25 +14,54 @@ import { ScrollArea } from '@/components/ui/scroll-area';
     markAsRead,
     markAllAsRead,
-    deleteNotification,
     loading
   } = useNotifications();
+  const navigate = useNavigate();
 
-  const handleMarkAsRead = (id: number, event: React.MouseEvent) => {
-    event.stopPropagation();
-    markAsRead(id);
+  // 根据通知内容判断跳转页面
+  const getNavigationPath = (notification: any) => {
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)